### DISCUSSION TITLE
give the fetchmail module the user option for "folders"

### DISCUSSION THREAD

**Discussion by strikegun79** [webmin/webmin#2104] [Ideas]
Hi,
is it possible to add into the fetchmail module a field to set the source folder of the mailbox?
I see that the fetchmail-lib.pl can handle the "folder" part. but it's missing in the save_poll.cgi and edit_poll.cgi to have that option in the GUI.

**Comment by jcameron** [comment]
That's a good idea! Let me look into this ...

**Comment by jcameron** [comment]
This has been implemented for inclusion in the next release...
